packages/meditrak-server/src/routes/GETSurveyGroups/assertSurveyGroupsPermissions.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import groupBy from 'lodash.groupby';
+import keyBy from 'lodash.keyby';
+import flattenDeep from 'lodash.flattendeep';
```

Comment:
Is there a particular reason to import like this rather than just `import { groupBy, keyBy, flattenDeep } from 'lodash';`?

I'm not entirely sure what the difference is and why you would prefer one style over the other. This is more just a question for me than anything.

Reply:
Great pickup @EMcQ-BES! Within the `meditrak-server` package we have the full `lodash` package available, whereas in other packages we specify the specific lodash utils individually in the `package.json`. @chris-bes you'll need to change to use the other format here (the others will only work "by chance", because they are being imported by other packages within the workspace, each library's code happens to be available)

I've made a wider task to clean this stuff up https://github.com/beyondessential/tupaia-backlog/issues/1205